_events/2018-10-12-GTN-cofest.md

```diff
@@ -0,0 +1,32 @@
+---
+site: freiburg
+tags: [training]
+title: GTN CoFest on Training Material
+starts: 2018-11-15
+ends: 2018-11-15
+organiser:
+  name: Galaxy Training Network
+  email: galaxy-training@lists.galaxyproject.org
+supporters:
+ - elixir
+ - denbi
+---
+
+The Galaxy Training Network has developed an infrastructure to deliver interactive training based on Galaxy: one central place ([https://training.galaxyproject.org](https://training.galaxyproject.org)) to aggregate training materials covering many current research topics. Each topic is supported by different tutorials developed and maintained by the community via a GitHub repository: [https://github.com/galaxyproject/training-material](https://github.com/galaxyproject/training-material).
+
+<div class="multiple-img">
+  <img src="/assets/media/gtn_cofests.png" style="max-width: 500px; margin: 20px auto;" alt="GTN cofest logo" />
+</div>
+
+The Galaxy Training Network is organizing regular online CoFests (Collaboration/Contribution Fests) every 3 months on the 3rd Thursday for a day of the collaborative work on the training content.
+
+**The next one will be on the 15th of November.** We plan to focus on genome annotation training; but anyone who would like to contribute on any other topics is very welcome to join.
+We will coordinate via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.459bpoqjjo780nm56q267ba7i8?authuser=1) (drop-in channel we will keep open the whole day) and [the GTN Gitter channel](https://gitter.im/Galaxy-Training-Network/Lobby).
```

Comment:
can you remove `?authuser=1` That will redirect to the user's second attached google account instead of their primary one which is a bit strange.